### PR TITLE
Re-add changelogs and add docs for CVE-2020-1736 reverts [2.9]

### PR DESCRIPTION
**Pull request by relrod**
##### SUMMARY

Re-add changelogs and add docs for CVE-2020-1736 reverts

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs

### CODE REVIEW

File: docs/docsite/rst/porting_guides/porting_guide_2.9.rst [hunk 1]
```diff
@@ -113,6 +113,10 @@ Writing modules
 
   .. seealso:: `The Python Relative Import Docs <https://www.python.org/dev/peps/pep-0328/#guido-s-decision>`_ go into more detail of how to write relative imports.
 
+* Version 2.9.12 of Ansible changed the default mode of file-based tasks to ``0o600`` when the user did not specify a ``mode`` parameter on file-based tasks. This was in response to a CVE report which we have reconsidered and no longer consider a flaw in Ansible. As a result, the ``mode`` change has been reverted in 2.9.13, and ``mode`` will now default to ``0o666`` as in previous versions of Ansible.
+* Affected modules included known_hosts, service, authorized_key, interfaces_file, pam_limits, pamd, redhat_subscription, selinux, and sysctrl. If you are using Ansible 2.9.12 and seeing new failures in these modules, upgrade to 2.9.13.
```
Comment (samdoran) [inline docs/docsite/rst/porting_guides/porting_guide_2.9.rst]: `sysctrl` --> `sysctl`

**Review by samdoran — COMMENTED** [review]
See review comments here https://github.com/ansible/ansible/pull/71516#pullrequestreview-478713418

File: changelogs/fragments/cve-2020-1736-revert.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+security_fixes:
+  - The fix for CVE-2020-1736 has been reverted and this CVE is no longer considered a flaw in ansible-base. Users are encouraged to specify a ``mode`` parameter in their file-based tasks when the files being manipulated contain sensitive data.
```
Comment (relrod) [inline changelogs/fragments/cve-2020-1736-revert.yml]:
```suggestion
  - The fix for CVE-2020-1736 has been reverted. Users are encouraged to specify a ``mode`` parameter in their file-based tasks when the files being manipulated contain sensitive data.
```

File: docs/docsite/rst/porting_guides/porting_guide_2.9.rst (hunk 1 above)
Comment (relrod) [inline docs/docsite/rst/porting_guides/porting_guide_2.9.rst]:
```suggestion
* If you changed any tasks to specify less restrictive permissions while using 2.9.12, those changes will be unnecessary (but will do no harm) in 2.9.13.
```

**Pull request merged**